src/installer/pkg/sfx/installers/dotnet-runtime-deps/dotnet-runtime-deps-debian.proj

```diff
@@ -1,15 +1,12 @@
 <Project Sdk="Microsoft.Build.NoTargets">
   <PropertyGroup>
-    <BuildDebPackage Condition="'$(TargetArchitecture)' == 'x64'">true</BuildDebPackage>
+    <BuildDebPackage>true</BuildDebPackage>
     <BuildRpmPackage>false</BuildRpmPackage>
   </PropertyGroup>
 
   <ItemGroup>
-    <LinuxPackageDependency Include="libc6;libgcc1;libgssapi-krb5-2;libstdc++6;zlib1g"/>
-    <LinuxPackageDependency Include="libssl1.0.0 | libssl1.0.2 | libssl1.1 | libssl3" />
-    <KnownLibIcuVersion Include="74;72;71;70;69;68;67;66;65;63;60;57;55;52" />
-    <LibIcuPackageDependency Include="libicu" Dependencies="libicu | @(KnownLibIcuVersion -> 'libicu%(Identity)', ' | ')" />
-    <LinuxPackageDependency
-      Include="@(LibIcuPackageDependency->Metadata('Dependencies'))" />
+    <LinuxPackageDependency Include="libc6;libgcc1;libgssapi-krb5-2;libstdc++6"/>
+    <LinuxPackageDependency Include="libssl3" />
+    <LinuxPackageDependency Include="libicu72" />
```

Comment:
Can we keep the old model for including the current and future `icu` versions? It prevents installation failures when distro updates their libicu package in a new minor (or major) release, after we ship the product. The alternative was to service the deps package, but that required prior knowledge of distro changes in their `icu` requirements - we did not always have it on time, which caused install issues until our next servicing release date.

Reply:
We weren't including future icu versions, we were adding support for the ones shipped by Ubuntu as they released.

Debian Sid still has ICU72, so I don't expect to need to react to an update for a bit.

I'll bring back the infrastructure for handling multiple versions, but I won't add back ICU74 as I think it's unlikely Debian will ever ship with that version (they'll likely jump further ahead when they eventually update).

Reply:
At one point I did add several versions that were higher than what was shipping in Ubuntu and Debian. But, if that isn't needed anymore, I'm ok with this plan.